app/views/data_access/retrieval.html.erb

```diff
@@ -14,9 +14,9 @@
   <nav class="data-access-nav">
     <a href="<%=edsc_path "/data/configure?#{@retrieval.source}" %>"><i class="fa fa-arrow-circle-o-left"></i> Back to Data Access Options</a>
   </nav>
-  <header class="data-access-header">
-    <h1>Success!</h1>
-    <p>Your request has been processed.  See below for information on accessing your data.</p>
+  <header class="data-access-header" data-bind="visible: ui.projectList.collectionsToDownload().length == 0 && ui.projectList.collectionOnly().length == 0 && ui.projectList.submittedOrders().length == 0 && ui.projectList.submittedServiceOrders().length == 0 && ui.projectList.collectionLinks().length == 0">
```

Comment:
@mightynimble I agree that it needs checking whether it disappears for those states - I'll get that done tomorrow.